main.go

```diff
@@ -12,13 +12,21 @@ import (
 
 func makeHandler(name string, fn func(http.ResponseWriter, *http.Request, string, string)) http.HandlerFunc {
```

Comment:
Are you sure `makeHandler` is used only for generating handlers that expect a name, followed by github username, then github repo.

It's not documented so it's hard to tell.